chain/client/src/client.rs

```diff
@@ -2140,3 +2151,52 @@ impl Client {
         }
     }
 }
+
+impl Client {
+    fn get_tier1_accounts(&mut self, tip: &Tip) -> Result<Arc<AccountKeys>, Error> {
```

Comment:
Could you add a comment here defining tier1_account is without referencing the network code? I think that's important for someone who's not familiar with the network code understand what this function should do.

Reply:
done